www/source/pricing/index.html.slim

```diff
@@ -0,0 +1,61 @@
+---
+layout: pricing
+classes: pricing
+inline_javascript: accordion()
+---
+.row
+  .columns.large-12.text-align-center
+    h1.text-white Pricing
+.pricing-row-wrapper
+  .background-waves-blue
+  .row.pricing-box-row
+    .columns.large-4.large-offset-4.text-align-center.pricing-box
+      .columns.large-12
+        h2.h5 Habitat Builder
+      .columns.large-12.price-entry
+        p.price.h2.mb-0 Free
+        p.small No credit card required
+      .columns.large-12.pricing-box-row
+        p.h6.mb-0 Unlimited application builds
+        p.mb-0 For both public and private packages
+      .columns.large-12.pricing-box-row.background-gray-light
+        p.h6.mb-0 Unlimited publishing
+        p.mb-0 Docker Hub only, more coming
+      .columns.large-12.pricing-box-row
+        p.h6.mb-0 Scaffoldings
+        p.mb-0 Ruby, Node, JVM, with more all the time
+      .columns.large-12.pricing-box-row.background-gray-light
+        p.h6.mb-0 Export your application to many formats
+        p.mb-0 Docker container, Application Container Image (ACI), AMI, Tar
+      .columns.large-12.pricing-box-row
+        p.h6.mb-0 Compatible with your favorite scheduler
+        p.mb-0 Kubernetes, CloudFoundry, Mesos DC/OS, Terraform, and more
```

Comment:
Terraform is a provisioner and not a scheduler. Should we drop it or alter the header here?